modules/home-manager/globals.nix

```diff
@@ -11,5 +11,7 @@
       default = "teal";
       description = "Global Catppuccin accent";
     };
+
+    enable = lib.mkEnableOption "Catppuccin globally";
```

Comment:
you should copy this to here too https://github.com/catppuccin/nix/blob/b87e5ee889aaeadc110fa8cf0c48e4115b39137c/modules/nixos/globals.nix#L2

Reply:
fixed